### DISCUSSION TITLE
Adding tracking of number of utilisation of software by glpi agent

### DISCUSSION THREAD

**Discussion by krenaudUR** [glpi-project/glpi-agent#308] [Ideas]
Hi, it's it possible that in the future the glpi agent counts the number of utilisation (number of start of a program) of software installed on the PC.

This solution is useful for an administrator to know which software is still in use or not.

**Comment by g-bougard** [comment]
Hi @krenaudUR 
this kind of feature is not really an inventory feature and need to use system monitoring to track programs starts... and even you need to associate program starts to inventory software... Very huge work. I don't think this will happen unless somebody is ready to invest a large budget.

**Reply by krenaudUR** [reply]
> Hi @krenaudUR this kind of feature is not really an inventory feature and need to use system monitoring to track programs starts... and even you need to associate program starts to inventory software... Very huge work. I don't think this will happen unless somebody is ready to invest a large budget.

Okay thanks for the reply
